repository/src/main/java/org/alfresco/repo/management/SafeApplicationEventMulticaster.java

```diff
@@ -152,6 +153,20 @@ public void removeApplicationListenerBean(String listenerBeanName)
         }
     }
 

```

Comment:
Remove // comment lines and make sure you add them to the commit message, so that is we really need to see why they were added it is there but does not clutter up the code.

Reply:
Removed comment (Added it to the commit message).